src/scheduler/batch/helpers.ts

```diff
@@ -71,49 +71,47 @@ function getUnpublishedDateContexts(today: Date): DateContext[] {
 
   // fill in the dates between 1 to 5 weeks from the todayDateString
   for (
```

Comment:
minor, no need to action now, but would a map be better? it looks like just an iteration through an array to produce a list.

Reply:
ah this one is incrementing by 7 (days) after each iteration until it reaches 35 (the five week window).  Probably a way to use a map to produce it but thought this was easier for now.  We can revisit but minor as you say.